src/drivers/spi/dw_spi/Mybuild

```diff
@@ -6,12 +6,13 @@ module dw_spi {
 	option number base_addr2 = 0
 	option number base_addr3 = 0
 
-	option number log_level = 0
+	option boolean spi_de0_nano_soc = false
```

Comment:
Do we really need this option? If so, then we need it for dw_i2c too, I think.

Reply:
There are several possible ways as we discussed:

* Use SPI option (as currently)
* Use GPIO module option like `spi_switch=<something>`, `spi_switch_enable=true`.
* Add a new module for the specific board configuration, which enables SPI or I2C.